app/models/collection.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# rubocop:disable Style/NumericPredicate
+class Collection < ApplicationRecord
+  def self.create_defaults
+    return if Collection.count > 0
+    # TODO: Define proper default collections, perhaps from DataSpace?
```

Comment:
Per our architecture diagram, I think we have four default collections:
1. Research Data
2. PPPL
3. ETDs
4. Library Resources